src/logger.rs

```diff
@@ -92,22 +145,7 @@ fn env_var_override(config: &mut LoggerConfig) {
     }
 }
 
-fn basic_dispatch(level_filter: LevelFilter) -> fern::Dispatch {
-    fern::Dispatch::new()
-        .level(level_filter)
-        .format(|out, message, record| {
-            out.finish(format_args!(
-                "[{level}][{target}] {message}",
-                level = record.level(),
-                target = record.target(),
-                message = message,
-            ))
-        })
-}
-
-fn colored_stdout() -> fern::Dispatch {
-    let color_config = fern::colors::ColoredLevelConfig::new();
-
+fn colored_stdout(color_config: fern::colors::ColoredLevelConfig) -> fern::Dispatch {
```

Comment:
This is also exposing fern types which I suppose isn't a huge deal but it would be awesome if you felt like fixing it ;)

Reply:
it's private

Reply:
Duh I'm sorry xD